package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "gulp build && node ./build/bin/www.js"
+    "start": "gulp build && node ./build/bin/www.js",
```

Comment:
`gulp build` ensures that `./build` exists. If `gulp build` fails, the script will short-circuit and the second part won't execute.